### ISSUE TITLE
Proxy Config Json Structure

### ISSUE THREAD

**Issue by helloanoop** [usebruno/bruno#684]
Below is the current structure we have for storing collection level proxy

```json
{
  "version": "1",
  "name": "bruno-testbench",
  "type": "collection",
  "proxy": {
    "use": true,
    "protocol": "http",
    "hostname": "localhost",
    "port": 4000,
    "auth": {
      "enabled": false,
      "username": "anoop",
      "password": "password"
    },
    "bypassProxy": ""
  }
}
```

The `use` can have values `global | true | false`
Initially, this field was called as `enabled` with values `true | false`

This change was made since the collection level proxy now has an additional option to use proxy setting defined at the app level in preferences (useful for folks who don't want to store proxy details at the collection level)

I propose to rename the `use` field back to `enabled`.
It can still have the values `global | true | false`

cc @phoval @mheidinger @mjhcorporate @mirkogolze

**Comment by mirkogolze** [comment]
Yes. I think the same property on collection and global level would be fine.
Currently on CLI-side still `enabled` is used to check the proxy on collection level.

**Comment by helloanoop** [comment]
This has been fixed and delivered in `v0.27.0`
